test/core/iomgr/resolve_address_test.cc

```diff
@@ -428,6 +428,70 @@ TEST_F(ResolveAddressTest, CancelWithNonResponsiveDNSServer) {
   PollPollsetUntilRequestDone();
 }
 
+// RAII class for pollset and pollset_set creation
+class PSS {
```

Comment:
Please call this something like `PollsetSetWrapper`.

https://google.github.io/styleguide/cppguide.html#General_Naming_Rules

Reply:
Done